Updated zones.json

avenmia opened the pull request:
I noticed when I tried to put in Cook, IL that the api returned an error because it was using the wrong zoneId. I saw the zoneId given in the zones.json was out of date so I curled https://api.weather.gov/zones and fed it into a new zones.json file. After uploading my version and selecting the Central Cook option from the updated list, the api call was working correctly.

Mubramaj commented:
Hi @avenmia, We just updated the master branch. Can you please update this pull request to be up to date with master and re-submit it. I will merge the pull request right away and deploy a new version of the app with your changes.

Thank you.

avenmia commented:
Hi @Mubramaj, I just updated it. Originally I had used the zones endpoint, but I believe the original used the zones/forecast/ endpoint so I changed the zones.json I submitted to use that one instead.

fabiendv commented:
Thank you for caching this :+1:
